packages/lab/src/static-list/StaticListItem.css

```diff
@@ -0,0 +1,18 @@
+.saltStaticListItem {
+  list-style: none;
+  all: unset;
+  color: var(--salt-content-primary-foreground);
+  display: flex;
+  gap: var(--salt-spacing-100);
+  align-items: baseline;
```

Comment:
Is this actually needed?

Reply:
Removed this. Added the style in icon example